client/me/purchases/manage-purchase/test/purchase-management-buttons.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { render, screen } from '@testing-library/react';
+import nock from 'nock';
+import { Provider as ReduxProvider } from 'react-redux';
+import { createReduxStore } from 'calypso/state';
+import ManagePurchase from '../index.jsx';
+
+const purchase = {
```

Comment:
Great work, upon running the tests, I am getting these console errors:

1. ![rrP5gk.png](https://user-images.githubusercontent.com/552016/209019197-367cc63e-2740-491d-813b-be43f612784a.png)
2. ![2uaRsD.png](https://user-images.githubusercontent.com/552016/209019265-3d373806-a3e3-4c93-b28a-1bd46ac7ca3c.png)
3. ![tM3qpN.png](https://user-images.githubusercontent.com/552016/209019324-941014ac-2dc8-45ac-a723-35ca5162cc0c.png)
4. ![Ic5URR.png](https://user-images.githubusercontent.com/552016/209019368-435764a1-f4a8-4c7d-9456-b35b30be375b.png)

Reply:
These are just proptypes warnings due to missing data in the mock Redux store. I've pushed a few commits to remove them.